.env.example

```diff
@@ -24,4 +24,15 @@ SOCIAL_AUTH_MICROMASTERS_LOGIN_URL=
 YOUTUBE_DEVELOPER_KEY=
 TIKA_SERVER_ENDPOINT=http://tika:9998/
 TIKA_CLIENT_ONLY=True
-WEBPACK_ANALYZE=True
\ No newline at end of file
+WEBPACK_ANALYZE=True
+
+SOCIAL_AUTH_OL_OIDC_OIDC_ENDPOINT=https://sso-qa.odl.mit.edu/realms/olapps
+OIDC_ENDPOINT=https://sso-qa.odl.mit.edu/realms/olapps
+SOCIAL_AUTH_OL_OIDC_KEY=ol-open-discussions-local
+SOCIAL_AUTH_OL_OIDC_SECRET=tpaVPFg4OAz0AVuQ61CGRVLqOGPP2v4L
+AUTHORIZATION_URL=https://sso-qa.odl.mit.edu/realms/olapps/protocol/openid-connect/auth
+ACCESS_TOKEN_URL=https://sso-qa.odl.mit.edu/realms/olapps/protocol/openid-connect/token
+USERINFO_URL=https://sso-qa.odl.mit.edu/realms/olapps/protocol/openid-connect/userinfo
+
+KEYCLOAK_BASE_URL=https://sso-qa.odl.mit.edu
+KEYCLOAK_REALM_NAME=olapps
```

Comment:
Can you add a README section on how to enable keycloak and set these values?  Also, a working `SOCIAL_AUTH_OL_OIDC_SECRET` shouldn't be included here, just noticed I got a github alert email about it.   Should probably change it to a new value now.